src/java.base/share/classes/javax/crypto/CryptoAllPermission.java

```diff
@@ -90,21 +91,20 @@ public int hashCode() {
     }
 
     /**
-     * Returns a new PermissionCollection object for storing
-     * CryptoAllPermission objects.
-     * <p>
+     * Returns a new {@code PermissionCollection} object for storing
+     * {@code CryptoAllPermission} objects.
      *
-     * @return a new PermissionCollection object suitable for
-     * storing CryptoAllPermissions.
+     * @return a new {@code PermissionCollection} object suitable for
+     * storing {@code CryptoAllPermissions} objects.
      */
     public PermissionCollection newPermissionCollection() {
         return new CryptoAllPermissionCollection();
     }
 }
 
 /**
- * A CryptoAllPermissionCollection stores a collection
- * of CryptoAllPermission permissions.
+ * A {@code CryptoAllPermissionCollection} stores a collection
+ * of {@code CryptoAllPermission} permissions.
```

Comment:
maybe change "permissions" to "objects" to be consistent with the rest of this file?

Reply:
Fixed.